src/Config/store.js

```diff
@@ -0,0 +1,4 @@
+import { createStore } from '@reduxjs/toolkit';
+import contactsReducer from '../Reducers/contacts';
+
+export default createStore(contactsReducer);
```

Comment:
change to configure store.

[https://redux-toolkit.js.org/tutorials/quick-start#create-a-redux-store](url)